CHANGES.rst

```diff
@@ -6,6 +6,13 @@ Changelog
 
 Release date: -
 
+0.5.2
+-----
+
+Release date: 2023/12/09
```

Comment:
No need to add a specific date here, I will update it when I make the new release.

```suggestion
Release date: N/A
```